app/component/map/tile-layer/Stops.js

```diff
@@ -129,6 +129,13 @@ class Stops {
     }
   };
 
+  shouldDrawStop = stop => {
+    return !(
+      stop.properties.type === 'CARPOOL' &&
+      !stop.properties.name.includes('P+M')
```

Comment:
Do you have an idea which IE versions we support? `String.includes` is only supported from version 12.

See https://caniuse.com/#feat=es6-string-includes